classes/asserter/exception.php

```diff
@@ -31,7 +31,17 @@ public function __construct(atoum\asserter $asserter, $message)
                 }
             }
 
-            $code = $test->getScore()->addFail($file, $class, $method, $line, get_class($asserter) . ($function ? '::' . $function : '') . '()', $message);
+            $asserterName = strtolower(get_class($asserter));
+
+            if ('mageekguy\atoum\asserters\\' === substr($asserterName, 0, 26)) {
```

Comment:
why not simply calling preg_replace here instead of a `if`+`substr`s ?

Reply:
Agree. Done.